cms/envs/common.py

```diff
@@ -143,7 +143,7 @@
 
     # If set to True, new Studio users won't be able to author courses unless
     # edX has explicitly added them to the course creator group.
```

Comment:
Update comment to say `Openedx Admin has added them to the course creator group`.  Are there docs somewhere of how to add a user to the course creator group?